pkgs/development/gnuradio-modules/nacl/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, mkDerivation
+, fetchFromGitHub
+, libsodium
+}:
+
+mkDerivation {
+  pname = "gr-nacl";
+  version = "2017-04-10";
+  src = fetchFromGitHub {
+    owner = "stwunsch";
+    repo = "gr-nacl";
+    rev = "15276bb0fcabf5fe4de4e58df3d579b5be0e9765";
+    sha256 = "018np0qlk61l7mlv3xxx5cj1rax8f1vqrsrch3higsl25yydbv7v";
+  };
+  disabled = ["3.8"];
```

Comment:
This also a point I'm not sure I chose the right terms and conditionals. In the future there should come another backwards incompatible Gnuradio version 3.9 where this list will need to be extended. These kind of gnuradio plugins will probably never get a new commit and perhaps there should be both a supported `disabled` list and `enabledOnly` variable? to be respected by `gnuradio.pkgs.mkDerivation`? The downside is that it'll increase the complexity there.

Reply:
Why not using boolean logic as is done with Python?

Reply:
You mean that this will be used inside packages' expressions:

```nix
  disabledIf = isGR3_8 or isGR3_9;
```

? It feels a bit better but I don't see how it is less complex..

Reply:
What is complex about it?

Reply:
I don't know whether using boolean logic or the current logic is more or less complex. Maybe either options is not complex.. I only expressed a slight dissatisfaction from the current situation, and I don't see a strong benefit in using boolean logic like Python over what's currently used. Maybe it's not that important.

Reply:
I finally chose using the attribute: `disabledForGRafter = "3.8"` which should not require a change once 3.9 will get in.